### PR TITLE
create NavBar component, add styled-components dependecies.

### PR DESCRIPTION
**Pull request by MilioFrankie**
### NavBar

- Run `yarn install` to get dependencies.

- Run `yarn start`

On chrome dev-tools, toggle through device screens to make sure navigation is responsive.

### CODE REVIEW

File: src/components/navbar/Styles.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    font-size: 1.75rem;
+  }
+
+  @media screen and (max-width: ${mobile}) {
+    font-size: 1rem;
+  }
+`;
+
+export const SubTitle = styled.h2`
+  margin: 0;
```
Comment (matelau) [inline src/components/navbar/Styles.js]: do you need margin 0? seems like the default is 0 according to https://stackoverflow.com/questions/40061984/default-value-of-margin-in-css

File: src/components/navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              </li>
+            ))}
+          </Nav>
+        </HeaderContent>
+      </header>
+    );
+  }
+}
+
+const title = { name: "DRY CREEK INDUSTRIES", subTitle: "General Contracting" };
```
Comment (matelau) [inline src/components/navbar/index.js]: IMO values you are initializing here should be at the top of the file, before any references.

Comment (matelau) [reply src/components/navbar/index.js]: If you want to make this component more reusable you should pass these in as props.

File: src/components/navbar/Styles.js (hunk 1 above)
Comment (MilioFrankie) [reply src/components/navbar/Styles.js]: Yeah, the Chrome browser adds a top and bottom margin.

File: src/components/navbar/index.js (hunk 2 above)
Comment (MilioFrankie) [reply src/components/navbar/index.js]: I can move them to the top, makes more sense to have it up there.

Comment (MilioFrankie) [reply src/components/navbar/index.js]: I was going to pass the links array down as props for my mobile nav. I guess it doesn't make sense why I made the title and subtitle as an object.

**Pull request closed without merging**